.github/workflows/check.yml

```diff
@@ -72,13 +72,22 @@ jobs:
     steps:
     - name: Install dependencies
       run: |
-        apt-get update
         apt-get update -y
         apt-get install -y software-properties-common
         add-apt-repository ppa:alexlarsson/flatpak
         apt-get install -y automake autoconf libtool gettext autopoint gcc \
           gtk-doc-tools shared-mime-info desktop-file-utils \
-          libglib2.0-dev libjson-glib-dev libfontconfig1-dev libfuse-dev
+          libglib2.0-dev libjson-glib-dev libfontconfig1-dev
+
+    - name: Install libfuse3
```

Comment:
It's already not possible to intsall xdg-desktop-portal as it is in current 16.04, in fact it uses a ppa for getting various dependencies. Not having control on that, I thought was just better to enable this in order to check glibc compatibility (that I think it's the main point of the test).

I've explained also it in https://github.com/flatpak/xdg-desktop-portal/pull/634#issuecomment-923515326

Reply:
It doesn't look like that PPA actually replaces any of the packages installed by this job in the workflow:

https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak?field.series_filter=xenial

My point was more that if we are requiring libraries not found in xenial, then perhaps it is also worth re-evaluating what the baseline should be.

Reply:
No, but these are still needed.

However, I'm not against it, I'd just wait maintainer decision on this, I'm open to everything.
The reason I added it, I found it useful, was for testing stuff such as [https://github.com/flatpak/xdg-desktop-portal/pull/634/commits/fa086800b11922f3294b7717875a3ee1b4a33023](`renameat2`) usage that wasn't supported otherwise.